backend/src/infrastructure/web/controllers/admin.controller.ts

```
@@ -235,13 +257,41 @@ export class AdminController {
+            kw.englishAudioUrl,
+            kw.japaneseAudioUrl,
+            index + 1,
+            new Date(),
+            new Date()
+          ));
+
+          await keywordRepository.createMany(keywordEntities);
+        }
+        
+        // Remove keywords from updates to avoid trying to save them to lesson
+        delete updates.keywords;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)